### PR TITLE
First try in making a table of contents

### PR DESCRIPTION
**Pull request by bedroesb**
This PR will bring a TOC to the tutorials that indexes H2 and H3 headings (can be changed):

Screenshot from 2021-05-26 14-36-28

- makes use of [jekyll-table-of-contents](https://github.com/ghiculescu/jekyll-table-of-contents)
- deleted some unused dependencies
- got rid of a java script error by loading the jquery in the head of the website
- used some simple css for now, can be changed

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -18,12 +18,14 @@
...
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+  <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js"></script>  
   <!-- Font Awesome -->
   <script defer src="/js/fontawesome-all.js"></script>
   {% if page.bioschemas %}
   <script type="application/ld+json">
      {{ page.bioschemas | jsonify }}
   </script>
   {% endif %}
+  <script src="{{ 'js/toc.js' | relative_url }}"></script>
```
Comment (bedroesb) [inline _includes/head.html:30]: to do: only there when layout == tutorial

**Comment by AlasdairGray** [comment]
@bedroesb when I click on a link in the ToC it jumps down but the heading is hidden below the navigation bar. Is there any way of ensuring that the heading stays on the screen?

**Review by AlasdairGray — CHANGES_REQUESTED** [review]
Looks nice. Can we add 'Table of Contents' as bold text at the top of the box?

**Comment by bedroesb** [comment]
@AlasdairGray I did a fix following this [tutorial](https://css-tricks.com/hash-tag-links-padding/) but it still feels a bit hacky, but it works though. I also took the freedom to change all the shadows to the same type used in the frontpage for consistency. I noticed that in mobile mode many things change including the fact that the title disappears and the navigation titles change.  For me it is confusing that in the normal website you don't have the word `tutorial` in the navigation, but you do have it in mobile mode. I could also not follow well the separation between the stylesheet.css and the other scss files. I think if you move everything over to the scss and make more use of the bootstrap classes, the code could be a lot simplified :) Just some thoughts! Sorry if I am commenting things that are already discussed in the past!

**Comment by AlasdairGray** [comment]
Thanks for the tweaks @bedroesb. Works well for `<h2>`, still not working well for `<h3>`, not sure why that should be the case since the changes you've made look like they should work.

@ljgarcia I've added a 'Table of Contents' title. Hopefully you approve of that.
Screen Shot 2021-06-02 at 13 11 12

**Comment by ljgarcia** [comment]
@AlasdairGray looks good. If all the changes you requested have been addressed (I did not check that in detail), please go ahead and merge this PR. Thanks

**Review by AlasdairGray — APPROVED** [review]
